scripts/install/deploy/install_db.sh

```diff
@@ -28,3 +28,39 @@ fi
 
 sudo cp timeless_pg.service /lib/systemd/system/
 sudo systemctl start timeless_pg.service
+
+#checks if the database exists and create if not
+if [! psql -lqt | cut -d \| -f 1 | grep -qw "timelessdb"; ]; then
+    echo "Creating database: timelessdb"
+    sudo -u postgres psql -c "CREATE DATABASE timelessdb;"
+fi
+
+#checks if the user exists and create if not
+if [! psql -t -c '\du' | cut -d \| -f 1 | grep -qw "timeless_user"; ]; then
+    echo "Creating user: timeless_user"
+    sudo -u postgres psql -c "CREATE USER timeless_user WITH
+        SUPERUSER
+        CREATEDB
+        CREATEROLE
+        INHERIT
+        LOGIN
+        ENCRYPTED PASSWORD 'timeless_pwd';"
+fi
+
+company = $(sudo -u postgres -H -- psql -d timelessdb -c "INSERT INTO company
+(name, code, address) values ('Timeless', 'Tm', '')")
+role = $(sudo -u postgres -H -- psql -d timelessdb -c "INSERT INTO role
+(name, works_on_shifts, company_id) values ('Administrator', False, $company)")
+password = "pass from rultor"
```

Comment:
@paulodamaso ``pass from rultor``? Shouldn't there be an env. variable here or something?

Reply:
@amihaiemil We should get this value from rultor, I'll have to research about it

Reply:
@paulodamaso ok, then maybe you can leave a puzzle for the research, or?